include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_merge.h

```diff
@@ -320,35 +361,61 @@ struct __parallel_merge_submitter_large<_IdType, _CustomName,
                         __start = __base_diagonals_sp_global_ptr[__diagonal_idx];
                     }
 
-                    __serial_merge(__rng1, __rng2, __rng3, __start.first, __start.second, __i_elem,
-                                   __nd_range_params.chunk, __n1, __n2, __comp);
+                    const auto __ends = __serial_merge(__rng1, __rng2, __rng3, __start.first, __start.second, __i_elem,
```

Comment:
The same: this local variable `__ends` may be unused.

Reply:
I believe a compiler just ignores it.

Reply:
It should cause warning, I think. Sure, depends on compiler options.

Reply:
But I would not duplicate the bulky line:
`__serial_merge(__rng1, __rng2, __rng3, __start.first, __start.second, __i_elem, __nd_range_params.chunk, __n1, __n2, __comp, __n);`
I dont know another way to avoid "possibility  variable unusing"...

Reply:
Option (1) - duplicate this one line under `if constexpr` condition check.
Option (2) - for example like described at https://stackoverflow.com/questions/1486904/how-do-i-best-silence-a-warning-about-unused-variables

Reply:
Add `[[maybe_unused]]` attribute: https://en.cppreference.com/w/cpp/language/attributes/maybe_unused